parm/wflow/verify_ens.yaml

```diff
@@ -21,6 +21,18 @@ default_task_verify_ens: &default_task_verify_ens
   queue: '&QUEUE_DEFAULT;'
   walltime: 01:00:00
 
+task_parse_vx_config_ens:
+  <<: *default_task_verify_ens
+  command: '&LOAD_MODULES_RUN_TASK_FP; "run_vx" "&JOBSdir;/JREGIONAL_PARSE_VX_CONFIG"'
```

Comment:
There is no `JREGIONAL_PARSE_VX_CONFIG` in the jobs directory. This is causing the `parse_vx_config_ens` task to fail.